src/util/metadata/side_metadata/spec_defs.rs

```diff
@@ -94,6 +94,10 @@ define_side_metadata_specs!(
     MS_LOCAL_FREE   = (global: false, log_num_of_bits: LOG_BITS_IN_ADDRESS, log_bytes_in_region: crate::policy::marksweepspace::native_ms::Block::LOG_BYTES),
     // First cell of thread free list in block for native mimalloc
     MS_THREAD_FREE  = (global: false, log_num_of_bits: LOG_BITS_IN_ADDRESS, log_bytes_in_region: crate::policy::marksweepspace::native_ms::Block::LOG_BYTES),
+    // Start and end marks by Compressor
+    COMPRESSOR_MARK = (global: false, log_num_of_bits: 0, log_bytes_in_region: LOG_MIN_OBJECT_SIZE as usize),
```

Comment:
You can say `LOG_BYTES_IN_WORD` to be more specific. You can also mention this in the comment above (1 bit per machine word).

Reply:
@no-defun-allowed Could you make this more explicit? Like Yi said, the min object size is subject to change. Just say `LOG_BYTES_IN_WORD`.